### ISSUE TITLE
Canary 9.4.5-canary.36 => 9.4.5-canary.37 - can't resolve main.js

### ISSUE THREAD

**Issue by followbl** [vercel/next.js#15261]
# Bug report

Sorry @timneutkens @Timer don't have time this morning for full-repro - but figured you'd want to know. 

Getting this just on simple build run between 36 => 37 canary

Wild guess - but could it be related to here? https://github.com/vercel/next.js/pull/15224/files#diff-4ec59a55144129a45f46da71c0777df9

```js
 ModuleNotFoundError: Module not found: Error: Can't resolve 'main.js' in '/vercel/419420aa'
```

## Describe the bug

Application won't build between the two versions

**Comment by timneutkens** [comment]
Kind of impossible to help based on this 😅 This doesn't even include a next.config.js

**Comment by followbl** [comment]
I'll close and re-open when I can give more info, figured the issue would be wide-spread, it's gotta be something with the `next-pwa`

**Comment by timneutkens** [comment]
If you can give a reproduction I can have a closer look 🙏

**Comment by followbl** [comment]
@timneutkens my hunch was correct - here you go https://github.com/followbl/pwa-bug

Steps to reproduce

1. run `yarn run build`

result

`ModuleNotFoundError: Module not found: Error: Can't resolve 'main.js' in`

expected result - no error on build

I'll report this in next-pwa incase there are any thoughts or helpful tips there. thank you as always!

**Comment by 7iomka** [comment]
Same error after upgrade to the latest canary version
![image](https://user-images.githubusercontent.com/18473137/87864089-74b6a800-c96c-11ea-9427-4459794979f5.png)

**Comment by timneutkens** [comment]
Fixed it in #15311 thanks for the reproduction!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
